lib/loader.js

```diff
@@ -19,6 +19,9 @@ module.exports = function(content, map) {
 	var sourceMap = query.sourceMap || false;
 	var resolve = createResolver(query.alias);
 
+  // suggest minimizing to user on production build
+  warnings.minimizeInProduction(query.minimize, process);
```

Comment:
What is `this` here? Does webpack already provides a warning API and I wrote all this code in vain? 😩😆